config/101-podsecuritypolicy.yaml

```diff
@@ -21,6 +21,7 @@ metadata:
     app.kubernetes.io/part-of: tekton-pipelines
   annotations:
     seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: runtime/default
```

Comment:
Does it need to be quoted ? (maybe just for the sake of consistency with the other annotations)

Reply:
For consistency, yes, it should be quoted.
Sorry, I missed that.